src/app/monitoring/monitoring.component.ts

```diff
@@ -7,9 +12,55 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MonitoringComponent implements OnInit {
 
-  constructor() { }
+  editors: Editor[];
+  editorSelectorItems: EditorSelectorItem[];
+  selectedEditor: Editor;
+  monitoringEditor: Editor;
+  selectedMedia: Media;
+  jobSuccess = false;
+  jobFailure = false;
+
+  constructor(private editorService: EditorService, private httpService: HttpService) {
+    this.editors = this.editorService.getEditors();
+    this.editorSelectorItems = this.editorService.getSelectorEditors();
+   }
 
   ngOnInit() {
   }
 
+  runEditorJob() {
+    this.initMessages();
+    if (this.selectedEditor) {
+      this.httpService.runEditorJob(this.selectedEditor.name)
+      .subscribe(
+        data => {
+          this.jobSuccess = true;
+        },
+        error => {
+          this.jobFailure = true;
+        }
+      );
+    }
+  }
+
+  runMediaJob() {
+    this.initMessages();
+    if (this.selectedMedia) {
+      this.httpService.runEditorJob(this.selectedMedia.name)
+        .subscribe(
+          data => {
```

Comment:
normalement tu as un warning qui te dis que la variable n'est pas utilisée. pour rémédier à ça tu dois nommer ta variable avec un underscore. 
```suggestion
          _data => {
          
```